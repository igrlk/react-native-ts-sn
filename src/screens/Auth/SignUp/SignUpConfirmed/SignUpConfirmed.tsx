import React from 'react';
import { View } from 'react-native';

import Screen from 'library/common/commonComponents/Screen';
import SuccessConfirmMessage from 'library/common/commonComponents/SuccessConfirmMessage';

import styles from './signUpConfirmedStyles';

export default function SignUpConfirmed() {
	return (
		<Screen isCentered>
			<View style={styles.container}>
				<SuccessConfirmMessage
					title='Confirmation sent'
					text='Please check your email for verfication'
				/>
			</View>
		</Screen>
	);
}
