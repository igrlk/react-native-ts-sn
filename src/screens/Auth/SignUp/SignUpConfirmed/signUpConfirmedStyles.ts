import EStyleSheet from 'react-native-extended-stylesheet';

export default EStyleSheet.create({
	container: {
		display: 'flex',
		alignItems: 'center',
		justifyContent: 'flex-start',
		marginTop: 40,
		marginBottom: 80,
	},
});
