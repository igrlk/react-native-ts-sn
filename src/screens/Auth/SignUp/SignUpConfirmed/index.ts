import SignUpConfirmed from './SignUpConfirmed';

export default SignUpConfirmed;
