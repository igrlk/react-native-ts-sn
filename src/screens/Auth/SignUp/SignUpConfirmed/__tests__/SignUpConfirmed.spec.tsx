import React from 'react';
import renderer from 'react-test-renderer';

import SignUpConfirmed from '../SignUpConfirmed';

describe('SignUpConfirmed', () => {
	it('match to snapshot', () => {
		const tree = renderer.create(<SignUpConfirmed />).toJSON();
		expect(tree).toMatchSnapshot();
	});
});
