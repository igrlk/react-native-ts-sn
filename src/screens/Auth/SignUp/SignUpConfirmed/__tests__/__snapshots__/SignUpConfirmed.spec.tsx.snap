// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SignUpConfirmed match to snapshot 1`] = `
<View
  style={
    Object {
      "flex": 1,
    }
  }
>
  <View
    style={
      Array [
        undefined,
        undefined,
        Object {},
      ]
    }
  >
    <View>
      <RCTScrollView>
        <View>
          <View>
            <View>
              <RNSVGSvgView
                align="xMidYMid"
                bbHeight="43"
                bbWidth="43"
                fill="none"
                focusable={false}
                height="43"
                meetOrSlice={0}
                minX={0}
                minY={0}
                style={
                  Array [
                    Object {
                      "backgroundColor": "transparent",
                      "borderWidth": 0,
                    },
                    undefined,
                    Object {
                      "opacity": 1,
                    },
                    Object {
                      "flex": 0,
                      "height": 43,
                      "width": 43,
                    },
                  ]
                }
                vbHeight={43}
                vbWidth={43}
                width="43"
              >
                <RNSVGGroup
                  fill={null}
                  fillOpacity={1}
                  fillRule={1}
                  font={Object {}}
                  matrix={
                    Array [
                      1,
                      0,
                      0,
                      1,
                      0,
                      0,
                    ]
                  }
                  opacity={1}
                  originX={0}
                  originY={0}
                  propList={
                    Array [
                      "fill",
                    ]
                  }
                  rotation={0}
                  scaleX={1}
                  scaleY={1}
                  skewX={0}
                  skewY={0}
                  stroke={null}
                  strokeDasharray={null}
                  strokeDashoffset={null}
                  strokeLinecap={0}
                  strokeLinejoin={0}
                  strokeMiterlimit={4}
                  strokeOpacity={1}
                  strokeWidth={1}
                  vectorEffect={0}
                  x={0}
                  y={0}
                >
                  <RNSVGPath
                    d="M21.5 0C9.64493 0 0 9.64493 0 21.5C0 33.3551 9.64493 43 21.5 43C33.3551 43 43 33.3551 43 21.5C42.9992 9.64493 33.3551 0 21.5 0ZM21.5 39.5346C11.5555 39.5346 3.46535 31.4445 3.46535 21.4992C3.46535 11.5548 11.5555 3.46458 21.5 3.46458C31.4445 3.46458 39.5346 11.5555 39.5346 21.5C39.5346 31.4445 31.4445 39.5346 21.5 39.5346Z"
                    fill={
                      Array [
                        0,
                        4281324438,
                      ]
                    }
                    fillOpacity={1}
                    fillRule={1}
                    matrix={
                      Array [
                        1,
                        0,
                        0,
                        1,
                        0,
                        0,
                      ]
                    }
                    opacity={1}
                    originX={0}
                    originY={0}
                    propList={
                      Array [
                        "fill",
                      ]
                    }
                    rotation={0}
                    scaleX={1}
                    scaleY={1}
                    skewX={0}
                    skewY={0}
                    stroke={null}
                    strokeDasharray={null}
                    strokeDashoffset={null}
                    strokeLinecap={0}
                    strokeLinejoin={0}
                    strokeMiterlimit={4}
                    strokeOpacity={1}
                    strokeWidth={1}
                    vectorEffect={0}
                    x={0}
                    y={0}
                  />
                  <RNSVGPath
                    d="M28.3535 14.6441L18.3179 24.6805L14.6457 21.0082C13.9694 20.3312 12.8725 20.3312 12.1962 21.0082C11.5192 21.6845 11.5192 22.7814 12.1962 23.4577L17.0936 28.355C17.4317 28.6931 17.8756 28.863 18.3187 28.863C18.7618 28.863 19.2049 28.6939 19.543 28.355L30.8037 17.0944C31.4799 16.4173 31.4799 15.3212 30.8037 14.6441C30.1267 13.9679 29.0305 13.9679 28.3535 14.6441Z"
                    fill={
                      Array [
                        0,
                        4281324438,
                      ]
                    }
                    fillOpacity={1}
                    fillRule={1}
                    matrix={
                      Array [
                        1,
                        0,
                        0,
                        1,
                        0,
                        0,
                      ]
                    }
                    opacity={1}
                    originX={0}
                    originY={0}
                    propList={
                      Array [
                        "fill",
                      ]
                    }
                    rotation={0}
                    scaleX={1}
                    scaleY={1}
                    skewX={0}
                    skewY={0}
                    stroke={null}
                    strokeDasharray={null}
                    strokeDashoffset={null}
                    strokeLinecap={0}
                    strokeLinejoin={0}
                    strokeMiterlimit={4}
                    strokeOpacity={1}
                    strokeWidth={1}
                    vectorEffect={0}
                    x={0}
                    y={0}
                  />
                </RNSVGGroup>
              </RNSVGSvgView>
              <Text>
                Confirmation sent
              </Text>
              <Text>
                Please check your email for verfication
              </Text>
            </View>
          </View>
        </View>
      </RCTScrollView>
    </View>
  </View>
</View>
`;
